### PR TITLE
[wx] Code refactoring concerning event handlers of menu items

### PR DESCRIPTION
**Pull request by rafaelx**

### CODE REVIEW

File: src/ui/wxWidgets/PasswordSafeFrame.cpp [hunk 1]
```diff
@@ -873,18 +939,7 @@ bool PasswordSafeFrame::Show(bool show)
   return wxFrame::Show(show);
 }
 
-/*!
- * wxEVT_COMMAND_MENU_SELECTED event handler for wxID_EXIT
- */
-
-void PasswordSafeFrame::OnExitClick( wxCommandEvent& /* evt */ )
```
Comment (dskrvk) [inline src/ui/wxWidgets/PasswordSafeFrame.cpp]: I can't seem to find where this got moved. Or was it dead code? The method is still registered to an event on line 129.

Same for some of the removed methods below.

**Comment by dskrvk** [comment]
Could you provide a summary of the change? Looks like some methods were moved from PasswordSafeFrame to menu handlers - anything else?

File: src/ui/wxWidgets/PasswordSafeFrame.cpp (hunk 1 above)
Comment (rafaelx) [reply src/ui/wxWidgets/PasswordSafeFrame.cpp]: All handlers related to 'File' menu items have been moved to the new implementation file 'MenuFileHandlers.cpp'. This corresponds to the Windows implementation and reduces the code at 'PasswordSafeFrame.cpp'.

- [win] MainEdit.cpp -> [wx] MenuEditHandlers.cpp
- [win] MainFile.cpp -> [wx] MenuFileHandlers.cpp
- [win] MainFilters.cpp -> [wx] Not implemented, yet.
- [win] MainManage.cpp -> [wx] MenuManageHandlers.cpp
- [win] MainMenu.cpp -> [wx] Not implemented.
- [win] MainView.cpp -> [wx] MenuViewHandlers.cpp

The entries in the event table of 'PasswordSafeFrame.cpp' have been rearranged to reflect the order of menu items as they are created. Same for the order of event handlers.

Includes are listed in the order of wx, core, application headers, C/C++ headers.

No functional changes.

Comment (rafaelx) [reply src/ui/wxWidgets/PasswordSafeFrame.cpp]: Now I see that the new file 'MenuFileHandlers.cpp' was not put under version control. This explains why so many lines of code have been removed compared to the number of added lines. Will fix it.

**Pull request merged**